app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1> Ingredients </h1>
+
+<% @ingredients.each do | ingredient | %>
+    <p><%= "#{ingredient.name}: #{ingredient.cost}" %></p>
+
+<% end %>
```

Comment:
Another way to do this is: `<p><%= ingredient.name %>: <%= ingredient.cost %></p>`.

What you have also works, but just want to show you a different approach.